### PR TITLE
Front: Remove "Ordering for company" from checkout if logged in

### PR DESCRIPTION
**Pull request by gurch101**
Only add company info form fields if user is not logged in
Ref SHOOP-2643

### CODE REVIEW

**Comment by tulimaki** [comment]
add to whitelist

**Comment by tulimaki** [comment]
ok to test

File: doc/changelog.rst [hunk 1]
```diff
@@ -142,6 +143,7 @@ Xtheme
 Classic Gray Theme
 ~~~~~~~~~~~~~~~~~~
 
+- Add Shoop Wishlist addon support for logged in users
```
Comment (shawnadelic) [inline doc/changelog.rst]: I think this was accidentally added from other PR

**Comment by shawnadelic** [comment]
Looks good other than the extra changelog entry. Will merge this once that is removed.

**Pull request closed without merging**